### PR TITLE
Adds test for Merchant Navigation

### LINKED ISSUE #72: User Story 4, Merchant Navigation
```
As a merchant user
I see the same links as a registered user
Plus the following links:
- a link to my merchant dashboard ("/dashboard")

Minus the following links/info:
- a link to my shopping cart ("/cart")
- the count of the items in my cart
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #75

depends on #73

### PR DESCRIPTION
**Pull request by Bradniedt**
-Added test for Merchant Navigation
-Added if else block to application.html.erb
-All tests passing

driver navigator brad/dan

closes #72

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -11,21 +11,36 @@
...
+            <li><%= link_to "Merchants", merchants_path %></li>
+            <li><%= link_to "Dashboard", dashboard_path %></li>
+            <li><%= link_to "Log Out", logout_path, method: :delete %></li>
+            <% end %>
+        <% else %>
+          <li><%= link_to "Home", root_path %></li>
+          <li><%= link_to "Items", items_path %></li>
+          <li><%= link_to "Merchants", merchants_path %></li>
+          <li><%= link_to "Cart", cart_path %></li>
+          <li>Total Items in Cart: <%= 0 %></li>
```
[the remark this answers is not in the document]
Comment (Bradniedt) [reply app/views/layouts/application.html.erb]: Agreed - the current code was just a product of having broke a lot of tests when we started adding the new Nav links. Let's look at this together this afternoon to come up with a DRY solution. Thanks!

File: config/routes.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
   get '/profile/orders', to: "orders#index"
   delete '/logout', to: "sessions#destroy"
   get '/merchants', as: :merchants, to: "users#index"
+  get '/dashboard', as: :dashboard, to: "users#show"
```
[the remark this answers is not in the document]
Comment (Bradniedt) [reply config/routes.rb]: Indeed - I think we'll need to get clarification from Dione since our story specifically called for the /dashboard path.

**Comment by asmolentzov** [comment]
Looks good! Discussed refactoring and DRY-ing up nav links with team, separated into own waffle card.
